tests/js/server/dump/dump-mixed.js

```diff
@@ -0,0 +1,553 @@
+/*jshint globalstrict:false, strict:false, maxlen:4000 */
+/*global assertEqual, assertNotEqual, assertTrue, assertFalse, assertNull */
+
+////////////////////////////////////////////////////////////////////////////////
+/// @brief tests for dump/reload
+///
+/// @file
+///
+/// DISCLAIMER
+///
+/// Copyright 2010-2012 triagens GmbH, Cologne, Germany
+///
+/// Licensed under the Apache License, Version 2.0 (the "License");
+/// you may not use this file except in compliance with the License.
+/// You may obtain a copy of the License at
+///
+///     http://www.apache.org/licenses/LICENSE-2.0
+///
+/// Unless required by applicable law or agreed to in writing, software
+/// distributed under the License is distributed on an "AS IS" BASIS,
+/// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+/// See the License for the specific language governing permissions and
+/// limitations under the License.
+///
+/// Copyright holder is triAGENS GmbH, Cologne, Germany
+///
+/// @author Jan Steemann
+/// @author Copyright 2012, triAGENS GmbH, Cologne, Germany
```

Comment:
2012 :P 
triAGENS :P

Reply:
I personally don't care, but you could fix all of those author statements in all files